### PR TITLE
Add product information when selecting single order

### PR DESCRIPTION
**Pull request by electriceeli**
Changes:
dev receives an array of products containing all product information when a single order is queried

To Test:
```run nodemon app.js```

In browser:
```localhost:8080/api/orders/(1-20)```

Expected results:
```
order {
  ...
  products: [
    ...
  ]
}
```

Resolves issues #10 

And I also added customer ID to order

Expected results :
``` correct ERD and tables```

### CODE REVIEW

**Review by melissabell456 — APPROVED** [review]
👍  Looks good! thanks for fixing the order data as well.

File: controllers/orders-ctrl.js [hunk 1]
```diff
@@ -10,9 +10,15 @@ module.exports.getOrders = (req, res, next) => {
...
   getSingleOrder(orderId)
-  .then( order => {
-    res.status(200).json(order)
-  })
+  .then( order => 
+    getOrdersProducts(orderId)
+    .then( arrayProducts => {
+      let joinedOrderProducts = { order };
+      joinedOrderProducts.order.products = arrayProducts
+
```
Comment (dhazlettjr) [inline controllers/orders-ctrl.js]: delete this line

**Review by dhazlettjr — APPROVED** [review]
💯

**Review by Valerie-Freeman — APPROVED** [review]
Everything looks great, results met expectations ✔️

**Pull request merged**